draft-labelling-describing.md

```diff
@@ -0,0 +1,207 @@
+Labeling and Describing
+
+## Introduction
+
+An accessible name is a name or a label used to identify the element for users of assistive technology. For example, a button’s accessible name can be "OK".
+
+An accessible description complements the accessible name with a description. An accessible description is always optional, and if present it does not need to be as brief as an accessible name.
+
+## Accessible name
```

Comment:
Introduce that you will be describing two different ways to get accessible names.